modules/ROOT/pages/_partials/usage-reports-tasks.adoc

```diff
@@ -0,0 +1,50 @@
+// tag::accessUsageReports[]
+
+. Sign in to Anypoint Platform using an account that has the Usage Viewer permission in your root organization.
+. In the Anypoint Platform navigation bar, click the business group selector, and ensure you are at the root organization level.
+. You can access usage reports either by:
+* Clicking your username icon and then clicking *Usage*, or
+* Selecting *Usage* in the Anypoint Platform navigation menu.
+. Select the product for which you want to view a usage report. 
++
+If you don't see the product listed, click *Switch View*. 
++
+. If applicable, click *Monthly Usage Report* or *Daily Usage Report*. 
++
+API Manager and API governance don't have daily usage reports.
+. Select the month or day of the usage report you want to view. 
++
+Only dates that Anypoint Platform collected data for the selected product are available.
++
+The usage report appears, displaying an assortment of usage metrics, which xref:usage-metrics.adoc[depend on the product you selected]. 
+
+
+// end::accessUsageReports[]
+
+//EDITORS: NO NEED TO REVIEW THE REST OF THIS FILE (THESE AREN'T BEING USED)
+// tag::accessUsageReportsLegacy[]
```

Comment:
This one is, tho.

Reply:
It's not being used in what's going to be published. This is another include within an include and the include it's used in isn't being used. At least not that portion of the file.